tests/support/unit/executor.ts

```diff
@@ -13,7 +13,9 @@ export function testProperty<E extends Executor = Executor, C extends Config = C
 ) {
 	assert.throws(() => { executor.configure(<any>{ [name]: badValue }); }, error);
 	executor.configure(<any>{ [name]: goodValue });
-	assert.equal(mockConsole.warn.callCount, 0, 'no warning should have been emitted');
+	for (let call of mockConsole.warn.getCalls()) {
```

Comment:
We should be passing something in to testProperty indicating that a deprecation warning (or any other console output) is expected.